DataFormats/L1TMuon/interface/EMTFOutput.h

```diff
@@ -70,7 +71,8 @@ namespace l1t {
     emtf::EventTrailer GetEventTrailer()    { return EventTrailer;  };
     emtf::MTF7Trailer GetMTF7Trailer()      { return MTF7Trailer;   };
     emtf::AMC13Trailer GetAMC13Trailer()    { return AMC13Trailer;  };
-    const uint64_t Dataword() const     { return dataword; };
+    const int Format_Errors() const { return format_errors; };
+    const uint64_t Dataword() const { return dataword; };
```

Comment:
@fwyzard:  because there were merge conflicts when merging 80x version of PR into 81x.